docs/docs/cmd/spo/listitem/listitem-add.md

```diff
@@ -30,6 +30,11 @@ m365 spo listitem add [options]
 
 --8<-- "docs/cmd/_global.md"
 
+## Remarks
+
+!!! tip "When using DateTime fields"
```

Comment:
Because of the last sentence, I suggest that we present this as a warning rather than a tip. If you provide wrong format, updating the field value will fail, so this is information is prerequisite for success.